### PR TITLE
Dockerise the fyle-intacct-app

### PR DESCRIPTION
**Pull request by labhvam5**

### CODE REVIEW

File: src/app/si/settings/general-mappings/general-mappings.component.html [hunk 1]
```diff
@@ -145,7 +145,9 @@
       <mat-form-field class="mappings-dialog--form-element" appearance="outline" required 
         *ngIf="generalSettings.sync_fyle_to_sage_intacct_payments">
         <mat-select placeholder="Select payment Account" formControlName="paymentAccount">
-          <mat-option *ngFor="let bankAccount of sageIntacctPaymentAccounts; let i = index" [value]="sageIntacctPaymentAccounts[i].destination_id">
+          <mat-option *ngFor="let bankAccount of sageIntacctPaymentAccounts; let i = index" [value]="sageIntacctPaymentAccounts[i].destination_id"
+          matTooltip="{{sageIntacctPaymentAccounts[i].value}}"  [matTooltipPosition] = "'right'"
+          >
```
Comment (Shwetabhk) [inline src/app/si/settings/general-mappings/general-mappings.component.html:150]: move this to line 149

File: Dockerfile.dev [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+# set working directory
+WORKDIR /app
+
+# add `/app/node_modules/.bin` to $PATH
+ENV PATH /app/node_modules/.bin:$PATH
+
+# install and cache app dependencies
+COPY package.json /app/package.json
+RUN npm install -g @angular/cli
```
Comment (ashwin1111) [inline Dockerfile.dev:16]: remove this off and check once, `angular/cli` is already present in the package.json file

File: .gitignore [hunk 3]
```diff
@@ -27,7 +27,7 @@ speed-measure-plugin*.json
 *.sublime-workspace
 
 # Environment files
-environment.localhost.ts
```
Comment (ashwin1111) [inline .gitignore:30]: Please don't remove this, else old folks would have a chance to commit this file.

Comment (labhvam5) [reply .gitignore:30]: ok

File: Dockerfile.dev (hunk 2 above)
Comment (labhvam5) [reply Dockerfile.dev]: done

**Review by Sravanksk — APPROVED** [review]

**Pull request merged**